lib/metadata_json_deps/forge_helper.rb

```diff
@@ -32,6 +32,30 @@ def get_metadata_json(name)
       metadata
     end
 
+    def get_metadata_json_uri(module_name)
```

Comment:
This method should be renamed to something like `get_module_data` as it returns more than just the metadata.

Reply:
Also since this method includes the metadata too, you could remove the old `get_metedata_json` method and instead cache the data retrieved from this one. Then you can also extract the `deprecated_at` field from this, removing the need for a separate call.